modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/MultiStepNav.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayMultiStepNav from '@clayui/multi-step-nav';
+import React from 'react';
+
+import Sheet from './Sheet';
+
+const MultiStepNav = ({steps}) => {
+	const currentStep = 0;
+
+	return (
+		<>
+			<ClayMultiStepNav>
+				{steps.map(({title}, index) => (
+					<ClayMultiStepNav.Item
+						active={currentStep === index}
+						complete={currentStep > index}
```

Comment:
Please, make sure the logic for `currentStep` will be implemented in the future because the currentStep variable will always be `0` in this context. So it won't work in this case.
```
complete={currentStep > index}
```

Reply:
yeah, we can do that when we'll be implementing the second step :smile: